docs/nidcpower/examples.rst

```diff
@@ -1,7 +1,9 @@
 Examples
 ========
 
-`You can download all nidcpower examples here <https://github.com/ni/nimi-python/releases/download/1.2.0/nidcpower_examples.zip>`_
+`You can download all nidcpower examples for master version here <https://github.com/ni/nimi-python/tree/master/src/nidcpower/examples>`_
```

Comment:
I would remove this, it will only cause confusion. Most clients don't know nor need to know that the code is in GitHub and there is a master version. Most of the time examples won't change anywhay between master and latest release.

Reply:
Removed